examples/file-sharing/src/network.rs

```diff
@@ -1,5 +1,5 @@
-use futures::channel::{mpsc, oneshot};
-use futures::prelude::*;
+use tokio::sync::{mpsc, oneshot};
```

Comment:
Not sure if it matters to others, but `futures` channels can still be used here.

Reply:
I can switch it back to `futures` if prefered